.env.defaults

```diff
@@ -30,6 +30,7 @@ YRBL_SAD_IMG_URL=https://cdn.lbryplayer.xyz/api/v3/streams/free/yrbl-sad/c2d9649
 ENABLE_COMMENT_REACTIONS=true
 ENABLE_FILE_REACTIONS=false
 ENABLE_CREATOR_REACTIONS=false
+TRUSTED_CHANNEL_MIN_STAKED_LEVEL=4
```

Comment:
I think this should be specific to video comments. There will be more features like this that use different levels.

Maybe `CHANNEL_STAKED_LEVEL_VIDEO_COMMENTS=4`

Then in the future we can have 
`CHANNEL_STAKED_LEVEL_OTHER_FEATURE`

Reply:
Please update the desktop config and set the level to 0
https://github.com/lbryio/lbry-desktop/blob/master/.github/workflows/deploy.yml#L76

Reply:
> I think this should be specific to video comments.

I blocked the videos in Post and Channel About as well.  Should I allow them?